package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "titlebar",
-  "version": "1.4.0",
+  "version": "1.5.0",
   "description": "Emulate OS X window title bar",
   "main": "index.js",
   "scripts": {
-    "demo-watch": "wzrd index.js:demo/index.js -- -s titlebar",
-    "demo-build": "browserify index.js -o demo/index.js -s titlebar"
+    "start": "wzrd index.js:demo/index.js -- -s titlebar",
+    "test": "browserify -t brfs index.js -o demo/index.js -s titlebar"
```

Comment:
Is it necessary to rename these? I don't think `-t brfs` is need, the package.json already contains the browserify transform field.